egs/aspire/s5/local/multi_condition/musan_prep_noise_list.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# Copyright 2016  Tom Ko
+# Apache 2.0
+# script to generate noise_list for MUSAN corpus
```

Comment:
mention that the output of this list is going to be used by your reverberate script, and the users can check that script for details about the format.